package.json

```diff
@@ -3,15 +3,20 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "^0.19.0",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-scripts": "3.0.1"
+    "react-router-dom": "^5.0.1",
+    "react-scripts": "3.0.1",
+    "react-transition-group": "^4.2.0",
+    "uuid": "^3.3.2"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "client": "npm start"
```

Comment:
Is this needed? Should you just be able to run `npm start` instead?

Reply:
It is not necessary, but I just rename it so that it matches with server command (npm run server). I will delete it tho!

Reply:
Yeah these commands are pretty standard across the community. Let's keep them as is so you can get used to them as well :)